### DISCUSSION TITLE
New stable relase ?

### DISCUSSION THREAD

**Discussion by momala454** [LizardByte/community#500] [Sunshine] [answered]
Hello,
thanks for the great tool. Seeing all the new beta releases, i'm wondering when you think a new stable will be out ?
Am I loosing a lot if I keep using stable ? the changelog is quite "hard" to read and full of dependency updates.

thanks

**Comment by peperunas** [comment]
Have you checked the pre-releases?

https://github.com/LizardByte/Sunshine/releases

**Comment by momala454** [comment]
That's what I was talking about with "beta" :)

**Reply by ReenigneArcher** [reply, marked as answer]
As we're all volunteering to maintain this project, we don't provide estimates.
